EKF/drag_fusion.cpp

```diff
@@ -45,12 +46,9 @@
 
 void Ekf::fuseDrag()
 {
-	float SH_ACC[4] = {}; // Variable used to optimise calculations of measurement jacobian
-	Vector24f H_ACC; // Observation Jacobian
-	float SK_ACC[9] = {}; // Variable used to optimise calculations of the Kalman gain vector
+	SparseVector24f<0,1,2,3,4,5,6,22,23> Hfusion;  // Observation Jacobians
 	Vector24f Kfusion; // Kalman gain vector
-	// TODO: resolve variance vs stdDev bug
-	const float R_ACC = _params.drag_noise; // observation noise variance in specific force drag (m/sec**2)**2
+	const float R_ACC = sq(_params.drag_noise); // observation noise variance in specific force drag (m/sec**2)**2
```

Comment:
Variables with the ending `_noise` are usually standard deviations, so taking the square of it for the variance is the right thing to do. But when adding the square, we should adapt the description and the default value of `drag_noise` in common.h, which is currently
`	float drag_noise{2.5f};			///< observation noise variance for drag specific force measurements (m/sec**2)**2
` to
`	float drag_noise{1.6f};			///< observation noise for drag specific force measurements (m/sec**2)
`

Reply:
Yes, that was a error carried over from previous. Will fix.

Reply:
Done